trade-finance-manager-ui/server/controllers/utilisation-reports/record-corrections/check-the-information/index.ts

```diff
@@ -15,7 +16,29 @@ export const getRecordCorrectionRequestInformation = async (req: Request, res: R
     const { reportId, feeRecordId } = req.params;
     const { user, userToken } = asUserSession(req.session);
 
-    const { bank, reportPeriod, correctionRequestDetails } = await api.getFeeRecordCorrectionRequestReview(reportId, feeRecordId, user._id, userToken);
+    const bankCorrectionRequestReviewResponse = await api.getFeeRecordCorrectionRequestReview(reportId, feeRecordId, user._id, userToken);
+
+    /**
+     * if record correction is already submitted
+     * should render page-not-found with custom reason
+     */
+    if (recordCorrectionRequestAlreadySubmitted(bankCorrectionRequestReviewResponse)) {
+      return res.render('utilisation-reports/page-not-found.njk', {
+        reason: 'The record correction request has been sent to the bank. You cannot make any changes to the request',
+        reportId,
+      });
```

Comment:
We have been using `ViewModel` types to define the shape of render params for different templates. It is good for type safety and understanding the interface between controllers and templates. I think it would be nice to have one for this.

Reply:
done